src/components/modal/GettingStartedModal.tsx

```diff
@@ -11,7 +11,7 @@ type GettingStartedModalProps = {
   imgAlt?: string;
 };
 
-const LOCAL_STORAGE_KEY = 'getting-started-modal';
+export const LOCAL_STORAGE_KEY = 'getting-started-modal';
```

Comment:
hacbs needs it's own key.
Let's prefix all hacbs localstorage keys with `hacbs/`

Reply:
updated in hacbs component